README.md

```diff
@@ -282,8 +282,11 @@ Actions supported:
 * `client.volumes.all()`
 * `client.volumes.find(id: 'id')`
 * `client.volumes.create(volume)`
+* `client.volumes.snapshots(id: 'id')`
+* `client.volumes.create_snapshot(id: 'id', name: 'snapshot-name')`
```

Comment:
Nope - it's the POST on /v2/volumes/:id/snapshots to create the snapshot itself.